### PR TITLE
Fixes for deployement in OS

### PR DESCRIPTION
**Pull request by ksneab7**

### CODE REVIEW

File: terraform_deploy/metadata_db.tf [hunk 1]
```diff
@@ -15,14 +29,20 @@ resource "aws_db_instance" "metadata_db" {
     instance_class                            = var.metadata_db_defaults["instance_class"]
     username                                  = var.metadata_db_defaults["username"]
     password                                  = local.db_password
-    parameter_group_name                      = var.metadata_db_defaults["parameter_group_name"]
+    parameter_group_name                      = length(aws_db_parameter_group.metadata_db_parameter_group) == 1 ? aws_db_parameter_group.metadata_db_parameter_group[length(aws_db_parameter_group.metadata_db_parameter_group)].name: ""
```
Comment (tyfarnan) [inline terraform_deploy/metadata_db.tf:32]: maybe indent to make more readable?

File: connectors/django/fma_django/models.py [hunk 2]
```diff
@@ -230,6 +237,27 @@ def delete_model_aggregation_schedule_remote(sender, instance, **kwargs):
...
+    """Checks the AWS Event Rule state for a given Federated Model.
+
+    :param federated_model: federated model instance
+    :type federated_model: An instance of a model object with an aggregation task.
+    :return: state of the rule associated with the aggregation task (disabled|enabled)
+    :rtype: (None|string)
+    """
+    state = None
+    try:
+        config = Config(connect_timeout=5, retries={"max_attempts": 0})
```
Comment (tyfarnan) [inline connectors/django/fma_django/models.py:250]: should we allow option to pass args for connect_timeout, retries, region?

Comment (ksneab7) [reply connectors/django/fma_django/models.py:250]: there is an issue created that abstracts out aws completely as it doesn't really belong in this file.
https://github.com/capitalone/federated-model-aggregation/issues/22

I would imagine that includes having a config file that accounts for this as well

Comment (JGSweets) [reply connectors/django/fma_django/models.py:250]: We could clarify the issue to reference that these can be settings that would be adjustable in the abstracted scenario.

**Review by JGSweets — APPROVED** [review]

File: connectors/django/fma_django/models.py [hunk 3]
```diff
@@ -99,7 +103,10 @@ def allow_aggregation(self):
         """
         if not self.scheduler:
             if self._allow_aggregation is None:
-                return False
+                allow_aggregation = get_aws_event_rule_state(self)
+                if allow_aggregation is None:
+                    return allow_aggregation
+                return get_aws_event_rule_state(self) == "enabled"
```
Comment (micdavis) [inline connectors/django/fma_django/models.py:109]:
```suggestion
                return allow_aggregation == "enabled"
```

**Review by micdavis — APPROVED** [review]

**Pull request merged**